components/common-data-items.js

```diff
@@ -164,18 +169,28 @@ export function SampleDataItems({ item, sources = null, children }) {
           </DataItemValue>
         </>
       )}
-      {item.sorted_fraction && (
+      {item.sorted_from && (
         <>
-          <DataItemLabel>Sorted Fraction</DataItemLabel>
+          <DataItemLabel>Sorted From Sample(s)</DataItemLabel>
```

Comment:
It looks like this always gets sorted from a single sample. Should this have the “(s)”?